packages/engine/src/audio/components/PositionalAudioComponent.ts

```diff
@@ -25,16 +25,19 @@ Ethereal Engine. All Rights Reserved.
 
 import { useEffect } from 'react'
 
-import { defineComponent, useComponent, useOptionalComponent } from '@etherealengine/ecs/src/ComponentFunctions'
-import { getMutableState, none, useHookstate } from '@etherealengine/hyperflux/functions/StateFunctions'
+import {
+  defineComponent,
+  removeComponent,
+  setComponent,
+  useComponent,
+  useOptionalComponent
+} from '@etherealengine/ecs/src/ComponentFunctions'
```

Comment:
for the ecs and hyperflux packages, prefer module level '@etherealengine/packagename' imports instead of deep imports